x-pack/plugins/global_search_providers/server/providers/saved_objects/map_object_to_result.test.ts

```diff
@@ -215,25 +216,31 @@ describe('mapToResults', () => {
 
     expect(mapToResults(results, typeRegistry, capabilities)).toEqual([
       {
+        icon: undefined,
         id: 'resultA',
         title: 'titleA',
         type: 'typeA',
         url: '/type-a/resultA',
         score: 100,
+        meta: { tagIds: [] },
       },
```

Comment:
Can you add a test where the object got tag + non-tag references to assert that `meta.tagIds` is correctly populated and that only `tag` references are included? (You can add a third, optional `references` parameter to `createObject`)